pkg/cli/install_test.go

```diff
@@ -0,0 +1,71 @@
+package cli
```

Comment:
These changes seem unrelated to memory requests and limits.

Reply:
Agreed. Removed it from this PR and will address it later in this one:

https://github.com/acorn-io/acorn/pull/1004